importexport/musicxml/exportxml.cpp

```diff
@@ -4977,8 +4977,9 @@ static void annotations(ExportMusicXml* exp, int strack, int etrack, int track,
                   wtrack = findTrackForAnnotations(e->track(), seg);
 
             if (track == wtrack) {
-                  if (commonAnnotations(exp, e, sstaff))
+                  if (commonAnnotations(exp, e, sstaff)) {
                         ;  // already handled
```

Comment:
why do we need this 'IF' at all?

Reply:
Good question, I guess mainly for documenting?

Reply:
427a5396

Reply:
@thibault ?